tests/system/api/test_secrets.py

```diff
@@ -187,3 +187,15 @@ def test_k8s_project_secrets_with_runtime(self):
 
         # Cleanup secrets
         self._run_db.delete_project_secrets(self.project_name, provider="kubernetes")
+
+    def test_file_to_project_secrets(self):
+        env_file = str(pathlib.Path(__file__).parent / "assets" / "envfile")
+        self._run_db.delete_project_secrets(self.project_name, provider="kubernetes")
+        mlrun.file_to_project_secrets(env_file, self.project_name)
```

Comment:
Better call with `set_env=False` to make sure we're not polluting the env for other tests.